assets/stylesheets/components/_modules.scss

```diff
@@ -0,0 +1,5 @@
+// contains each self-contained component in its own .scss partial
+
+@import 'alerts',
+        'tables',
+        'ig-confirmation';
```

Comment:
What does `ig-confirmation` stand for? See my previous note about naming conventions.